templates/patterns/list_group/pattern-list-group.html.twig

```diff
@@ -1,12 +1,24 @@
 {#
 /**
  * @file
- * List group component.
+ * List group pattern.
  */
 #}
+{% set _items = [] %}
+
+{% for item in items %}
+  {% if item.html %}
+    {% set item = item|merge({
+      'variant': item.link_style
+    }) %}
+  {% endif %}
+  {% set _items = _items|merge([item]) %}
+{% endfor %}
```

Comment:
I think this is wrong, we are not setting an API here. Any change in the component will change the pattern, which is not good.